scripts/enum/detects.lua

```diff
@@ -5,14 +5,15 @@ xi = xi or {}
 
 xi.detects =
 {
-    NONE        = 0x000,
-    SIGHT       = 0x001,
-    HEARING     = 0x002,
-    LOWHP       = 0x004,
-    NONE1       = 0x008,
-    NONE2       = 0x010,
-    MAGIC       = 0x020,
-    WEAPONSKILL = 0x040,
-    JOBABILITY  = 0x080,
-    SCENT       = 0x100,
+    NONE              = 0x000,
+    SIGHT             = 0x001,
+    HEARING           = 0x002,
+    SIGHT_AND_HEARING = 0x003,
```

Comment:
I initially hated this when I first saw it, but then I remember some annoyance recently having to do `bit.band(xi.detects.SIGHT, xi.detects.HEARING)` recently - so this is a good choice I think 👍